content/blog/2019/07/2019-07-03-remoting-kafka-kubernetes-phase-1.adoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "Remoting over Apache Kafka plugin with Kafka launcher in Kubernetes"
+tags:
+- remoting
+- kafka
+- kubernetes
+- helm
+- developer
+- gsoc
+- gsoc2019
+author: longnguyen
+---
+
+= Remoting over Apache Kafka plugin with Kafka launcher in Kubernetes
+
+== Introduction
```

Comment:
Would be nice to have a quick intro for the Remoting over Apache Kafka project (e.g. few sentences reference blogposts by @pvtuan10)